src/ontology/hp-edit.owl

```diff
@@ -211687,17 +211687,11 @@ AnnotationAssertion(dcterms:creator <http://purl.obolibrary.org/obo/HP_4000079>
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/HP_4000079> "Sensory seeking")
 SubClassOf(<http://purl.obolibrary.org/obo/HP_4000079> <http://purl.obolibrary.org/obo/HP_5200046>)
 
-# Class: <http://purl.obolibrary.org/obo/HP_4000080> (Reduced social initiation)
-
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:18064552") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_4000080> "Failure of a person to initiate social interactions with others. For a social interaction to have been initiated by someone, there must have been an absence of social interactions from either party for a period. The interaction must also be unprompted by another party.")
-AnnotationAssertion(dc:date <http://purl.obolibrary.org/obo/HP_4000080> "2022-03-02T16:47:44Z"^^xsd:dateTime)
-AnnotationAssertion(dcterms:contributor <http://purl.obolibrary.org/obo/HP_4000080> <https://orcid.org/0000-0001-9112-5148>)
-AnnotationAssertion(dcterms:contributor <http://purl.obolibrary.org/obo/HP_4000080> <https://orcid.org/0000-0002-4142-7153>)
-AnnotationAssertion(dcterms:contributor <http://purl.obolibrary.org/obo/HP_4000080> <https://orcid.org/0000-0002-6670-9157>)
```

Comment:
why remove provenance information when obsoleting a term?

Reply:
Simplified explanation: the same contributors were present in the term they were merged to. This one exception also had a 'creator', which I added to the contributor list. Because the terms were added via a ROBOT template--they do not have creator or date.

Reply:
thanks both for the explanation!